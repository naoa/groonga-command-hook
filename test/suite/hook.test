register commands/hook

table_create Entries TABLE_HASH_KEY ShortText
column_create Entries title COLUMN_SCALAR ShortText
column_create Entries content COLUMN_SCALAR ShortText

table_create Tokens TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
column_create Tokens entries_title COLUMN_INDEX|WITH_POSITION Entries title
column_create Tokens entries_content COLUMN_INDEX|WITH_POSITION Entries content

load --table Entries
[
{"_key": "mroonga", "title": "groonga and MySQL", "content": "groonga + MySQL = mroonga."}
]

hook_status --table Entries --column title
hook_delete --table Entries --column title --no_target_only false
