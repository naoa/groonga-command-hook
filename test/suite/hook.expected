register commands/hook
[[0,0.0,0.0],true]
table_create Entries TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Entries title COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
column_create Entries content COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Tokens TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
[[0,0.0,0.0],true]
column_create Tokens entries_title COLUMN_INDEX|WITH_POSITION Entries title
[[0,0.0,0.0],true]
column_create Tokens entries_content COLUMN_INDEX|WITH_POSITION Entries content
[[0,0.0,0.0],true]
load --table Entries
[
{"_key": "mroonga", "title": "groonga and MySQL", "content": "groonga + MySQL = mroonga."}
]
[[0,0.0,0.0],1]
hook_status --table Entries --column title
[[0,0.0,0.0],[1,["(nil)"]]]
hook_delete --table Entries --column title
[[0,0.0,0.0],0]
